### ISSUE TITLE
Capacitors Front Board

### ISSUE THREAD

**Issue by brendan1495** [tianfeng33/O-MVS#5]
Hi could you tell me the rating of capacitors C1 & C2 on the front board the bom just says 1210 x7r

**Comment by tianfeng33** [comment]
Yeah anything over 5v is fine and with ceramics you are probably getting something way above that likely

**Comment by brendan1495** [comment]
Thanks a lot

**Comment by Vanles** [comment]
Does the capacitance not matter?

Can you give me an example of what I could use?
